mlir/utils/jenkins/Jenkinsfile.downstream

```diff
@@ -18,11 +18,16 @@ pipeline {
                 sh '''
                     # install pkg dependencies
                     sudo apt update
-                    sudo apt install -y ninja-build wget git python cmake
+                    sudo apt install -y ninja-build wget git python3.7 cmake python3-pip
+		    if [ -f /usr/bin/python ]; then
+                      	/bin/rm /usr/bin/python
+                    fi 
+		    ln -s /usr/bin/python3.7 /usr/bin/python
```

Comment:
This is wrong. Directly removing the binary is very hacky and can confuse the package manager.

Choose one of the following ways:
 - `alias python=python3`
 - Remove the python package before installation of python3.7

Reply:
I removed a symlink

Reply:
Same concept applies for symlink. The link must be created by **some package**, likely python. Now you are removing a simlink of the package without actually removing the entire package. This creates an edge case for package manager to handle.

Reply:
This should be Ok. I didn't install python. (Docker automatically installed python3.6. ) Using an alias or removing the python package wouldn't help because python is not installed.  Alternatively, I may be able to use the old version of subprocess() to avoid installation of python3.7, as the reason why I wanted to have python3.7 was to make two subprocess() calls happy.  I tried the old version of subprocess() and that worked in my local machine, but the build failed at Jenkins. I will take a look.